web/js/helpers/OTHelper.js

```diff
@@ -11,7 +11,7 @@
   // in IE dynamic loading the library doesn't work. For the time being, as a stopgap measure,
   // loading it statically.
   if (dynamicOTLoad) {
-    var OPENTOK_API = 'https://static.opentok.com/webrtc/v2/js/opentok.min.js';
+    var OPENTOK_API = 'https://enterprise.opentok.com/v2/js/opentok.min.js';
```

Comment:
Why are we moving to enterprise?
If our instance is now enterprise we should set this up to be configurable since this is an opensource demo code.